drivers/serial/uart_rtt.c

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2019 omSquare s.r.o.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/uart.h>
+#include <SEGGER_RTT.h>
+
+struct uart_rtt_config {
+	void *up_buffer;
+	size_t up_size;
+	void *dw_buffer;
```

Comment:
`dw_` is not very common abbreviation for `down`. I would stick with `down_buffer`

Reply:
Ok, noted.